src/components/votes/votesTab.css

```diff
@@ -0,0 +1,140 @@
+@import '../app/variablesV2.css';
+
+.wrapper {
+  margin-bottom: 40px;
+  overflow: initial;
+
+  & header {
+    justify-content: space-between;
+  }
+
+  & .filterHolder {
+    height: 32px;
+    width: 200px;
+
+    & input {
+      font-size: var(--button-font-size-s);
+      height: 100%;
+    }
+  }
+
+  & .lastHeading {
+    & :global(.tooltip-window) {
+      left: auto;
+      right: -12px;
+      transform: translateX(0);
+
+      & :global(.tooltip-arrow) {
+        right: 16px;
+        left: auto;
+      }
+    }
+  }
+}
+
+.empty {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  font-family: var(--heading-font);
+  font-size: var(--paragraph-font-size-s);
+  font-weight: var(--font-weight-bold);
+  justify-content: center;
+  line-height: 1.5;
+  text-align: center;
+  color: var(--color-content-light);
+}
+
+.results {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  margin: 0;
+  min-height: 300px;
+  overflow: hidden;
+  position: relative;
+
+  &.hasMore {
+    padding-bottom: 30px;
+  }
+
+  &.isLoading {
+    & .loadingOverlay {
+      background: rgba(255, 255, 255, 0.8);
```

Comment:
maybe you can use a constant in here

Reply:
This one is because it's just for the mask when in the loading state, that we will probably change for the final design.

Reply:
good